go/networkqualityd.go

```diff
@@ -67,9 +84,29 @@ func setCors(h http.Header) {
 	h.Set("Access-Control-Allow-Headers", "*")
 }
 
+type handlers struct {
+	enableCORS bool
+}
+
+// BulkHandlers returns path, handler tuples with the provided prefix.
+func BulkHandlers(prefix string, enableCORS bool) map[string]http.HandlerFunc {
```

Comment:
`bulkHandlers`?

Reply:
This is for others adding into their own handlers (I'm working with groups who just want the data generation and not the rest of this).

The future here will be to split into package and command driver for the server.